thijsvanloef/palworld-server-docker
# Add player logging

## Context 

* Currently the game does not announce when players join or leave nor is it logged

## Choices

* Adding in logging as a background task

## Test instructions

1. Start a server
2. Verify it logs it you leaving and joining in docker
3. With players in game test with others joining
4. Test multiple players joining within the same polling period
5. Delete player data, create a new players with any name that may break it (Spaces, commas, numbers, etc)

Output will look like
```
palworld-server  | Dashboy1998 has joined
palworld-server  | Broadcasted: Dashboy1998_has_joined
palworld-server  | Dashboy1998 has left
palworld-server  | Broadcasted: Dashboy1998_has_left
```

If #425 has been merged then the output will be
```
palworld-server  | Broadcasted: Dashboy1998_has_joined
palworld-server  | Broadcasted: Dashboy1998_has_left
```

## Checklist before requesting a review
